Lib/booleanOperations/booleanGlyph.py

```diff
@@ -78,7 +82,7 @@ def _get_clockwise(self):
 
     def _get_bounds(self):
         if self._bounds is None:
-            from robofab.pens.boundsPen import BoundsPen
+            from fontTools.pens.boundsPen import BoundsPen
```

Comment:
the docstring of the robofab ControlBoundsPen says:
```
This replaces fontTools/pens/boundsPen (temporarily?)
The fontTools bounds pen takes loose anchor points into account, this one doesn't
```
And it also has an additional `_addMoveTo` method. I'm looking at it now.